releasenotes/notes/add-ipv6-support-for-hostnames-0fb700a1ef7ed57e.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
```

Comment:
```suggestion
features:
```

I think this is a feature rather than a fix since we had not stated support for or tested IPv6 addresses

Reply:
Sounds good, updated